cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -178,8 +229,109 @@ public static Double getSizeInMiB(String size) {
      * @param throughput The String representation of the throughput.
      * @return The equivalent number of kibibytes.
      */
-    public static Double getThroughputInKiB(String throughput) {
+    public static String getThroughputInKiB(String throughput) {
         String size = throughput.substring(0, throughput.indexOf("B"));
-        return StorageUtils.convertTo(size, "Ki");
+        return String.valueOf(StorageUtils.convertTo(size, "Ki"));
+    }
+
+    private void processCapacityEntries(BrokerCapacity bc, Storage s) {
```

Comment:
This is a nit, but I really don't like short acronyms as variable names. It makes it hard to figure out what they are later on.